tests/test_float32.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+# pylint: disable=missing-docstring
+import unittest
+
+import hypothesis
+import hypothesis.strategies
+
+import numenc_module
+
+MIN_FLOAT32 = -3.402823466385288598117041834851e+38
+MAX_FLOAT32 = 3.402823466385288598117041834851e+38
+
+
+class TestEncodingFloat32(unittest.TestCase):
+    """ Tests encoding and decoding of float 32"""
+
+    @hypothesis.given(hypothesis.strategies.floats(min_value=MIN_FLOAT32, max_value=MAX_FLOAT32, width=32))
+    def test_encode_decode_automatic(self, value: float):
+        hypothesis.assume(value)
+        self.assertEqual(value, numenc_module.to_float32(numenc_module.from_float32(value)))
+
+    def test_encode_order_corner_cases(self):
+        # yapf: disable
+        table_smaller = [
+            (-0.0, float('inf'), "-0 should be smaller than infinity "),
+            (+0.0, float('inf'), "0 should be smaller than infinity "),
+            (float('-inf'), -0.0, "negative infinity should be smaller than -0"),
+            (float('-inf'), +0.0, "negative infinity should be smaller than +0"),
+            (1.0, float('inf'), "1 should be smaller than infinity"),
+            (float('-inf'), 1.0, "negative infinity should be smaller than 1"),
+            (-1992.1210, -0.0, "-1992.1210 should be smaller than -0"),
+            (-0.0, 1231111.123034, "-0 should be smaller than 1231111.123034"),
+            (0.0, 1231111.123034, "0 should be smaller than 1231111.123034"),
+        ]
+
+        table_equals = [
+            (float('inf'), float('inf'), "infinity should be equals to itself"),
```

Comment:
should equal itself? Please see on Internet what is correct English and also change the descriptions below.